orderfile/app/build.gradle

```diff
@@ -0,0 +1,60 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdk 31
+
+    defaultConfig {
+        applicationId "com.example.orderfiledemo"
+        minSdk 21
+        targetSdk 31
+        versionCode 1
+        versionName "1.0"
+        ndkVersion "23.1.7779620"
```

Comment:
Should use `25.2.9519653` for new samples (that's probably worth retesting so I'm not committing this directly)

Reply:
Actually, since this is redefined below, does this even do anything? One of the two has no effect but I'm not sure which.

For that matter, if you don't care which version you use, just delete the line and let gradle decide.